pkgs/by-name/an/ananicy-cpp/package.nix

```diff
@@ -16,7 +16,7 @@
   withBpf ? true,
 }:
 
-clangStdenv.mkDerivation (finalAttrs: {
+llvmPackages.stdenv.mkDerivation (finalAttrs: {
```

Comment:
what's wrong with leaving it as `clangStdenv` ?

Reply:
Nothing, for some reason I thought `clangStdenv` was an alias of `llvmPackages.stdenv`.